client/flags/flags.go

```diff
@@ -111,6 +114,20 @@ func AddTxFlagsToCmd(cmd *cobra.Command) {
 	cmd.SetOut(cmd.OutOrStdout())
 }
 
+// GetKeyringDir finds the directory for the keyring storage
+func GetKeyringDir(pflags *pflag.FlagSet, homeDir string) (string, error) {
+	keyringDir, err := pflags.GetString(FlagKeyringDir)
```

Comment:
```suggestion
	keyringDir, _ := pflags.GetString(FlagKeyringDir)
```
You can ignore the error